plugins/parsers/json_v2/README.md

```diff
@@ -7,6 +7,9 @@ this playground to test out your GJSON path here:
 [gjson.dev/](https://gjson.dev). You can find multiple examples under the
 [`testdata`][] folder.
 
+**NOTE:** This parser has a different semantics comparing to regular JSON Parser - if you don't have any configuration fields or tags it will not produce any metric on contrary to JSON Parser.
+Also it cannot be used as an output format.
```

Comment:
This is about a parser, output formats are defined by the existing serialisers..
```suggestion
```